code/src/index.css

```diff
@@ -1,13 +1,120 @@
+*{
+  overflow-x: hidden;
+}
```

Comment:
I'm assuming that this piece of code was meant to hide any overflow outside of the posts-box for long words. Right now it clips the word and hides everything that doesn't fit. A suggestion would be to add word-wrap: break-word; instead.